tests/app/test_app.py

```diff
@@ -3,9 +3,10 @@
 import logging
 
 import pytest
+from _pytest.monkeypatch import MonkeyPatch
```

Comment:
You can import this from `pytest`:

https://github.com/pytest-dev/pytest/blob/4f3f36c396b52f8398bc4734ff0c00c57cf1fed1/src/pytest/__init__.py#L122

Reply:
Yeah, I don't know why PyCharm defaults to this import :shrug: